storage/database/src/main/java/de/danoeh/antennapod/storage/database/DBWriter.java

```diff
@@ -685,65 +685,28 @@ public static Future<?> resetPagedFeedPage(Feed feed) {
         });
     }
 
-    /*
-     * Sets the 'read'-attribute of all specified FeedItems
-     *
-     * @param played  New value of the 'read'-attribute, one of FeedItem.PLAYED, FeedItem.NEW,
-     *                FeedItem.UNPLAYED
-     * @param itemIds IDs of the FeedItems.
-     */
-    public static Future<?> markItemPlayed(final int played, final long... itemIds) {
-        return markItemPlayed(played, true, itemIds);
-    }
-
-    /*
+    /**
      * Sets the 'read'-attribute of all specified FeedItems
      *
-     * @param played  New value of the 'read'-attribute, one of FeedItem.PLAYED, FeedItem.NEW,
-     *                FeedItem.UNPLAYED
-     * @param broadcastUpdate true if this operation should trigger a UnreadItemsUpdate broadcast.
-     *        This option is usually set to true
-     * @param itemIds IDs of the FeedItems.
+     * @param played  New value of the 'read'-attribute, one of FeedItem.PLAYED, FeedItem.NEW, FeedItem.UNPLAYED
+     * @param items   to be marked.
      */
-    public static Future<?> markItemPlayed(final int played, final boolean broadcastUpdate,
```

Comment:
Before we could choose if the event should be broadcast. Now it always is. Is this a problem? Where did we use it without broadcasting? In a loop where we didn't want the app to reload lists repeatedly?